tests/test_cora_transform.py

```diff
@@ -415,8 +415,8 @@ def test_ukis_pdf(self):
         pages = list(PackerTests.extract_text(path))
         self.assertEqual(2, len(pages))
         questions = [
-            ("3.3", "3.5", "3.8", "3.10", "3.12", "3.14", "3.17", "11.1"),
-            ("11.2", "11.3", "13.1")
+            ("3.3", "3.5", "3.8", "3.10", "3.12", "3.14"),
```

Comment:
Did we work out why these questions moved tuples?

Reply:
So, the first tuple is on the first page of the pdf and the second on the second, I was just thinking about whether this might break something if it needs to be specific? 
If I run it with the question in the first I get this: `AssertionError: unexpectedly None`